crates/nu-command/src/experimental/is_admin.rs

```diff
@@ -48,3 +47,44 @@ impl Command for IsAdmin {
         ]
     }
 }
+
+#[cfg(unix)]
+fn is_root() -> bool {
+    nix::unistd::Uid::current().is_root()
+}
+
+#[cfg(windows)]
+fn is_root() -> bool {
+    use windows::Win32::{
+        Foundation::{CloseHandle, FALSE, INVALID_HANDLE_VALUE},
+        Security::{GetTokenInformation, TokenElevation, TOKEN_ELEVATION, TOKEN_QUERY},
+        System::Threading::{GetCurrentProcess, OpenProcessToken},
+    };
+
+    let mut handle = INVALID_HANDLE_VALUE;
+    let mut elevated = false;
+
+    unsafe {
```

Comment:
just to track the change suggested by @sholderbach, maybe add some comment here to explain the `unsafe` block :yum:

Reply:
@nibon7 

i'm actually confused by where the functions you call in the `unsafe` block come from :open_mouth: 

i see you have basically added the dosctring of these functions, but this should be information provided by some documentation, not by our code base, right?

i personally thought more about a few shorts sentences explaining *why* we need the `unsafe` block :yum: